### PR TITLE
15863 moved radio buttons in approval type component closer together …

### PR DESCRIPTION
**Pull request by PaulGarewal**
…vertically

*Issue #:* /bcgov/entity#15863

*Description of changes:*

- Shifted the radio buttons in Approval type vertically to be closer together. This will make the spacing more proportional to the new dividing line. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/approval-type/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bcrs-shared-components/approval-type",
-  "version": "1.0.31",
+  "version": "1.0.32",
```
Comment (JazzarKarim) [inline src/components/approval-type/package.json:3]: Paul, in the shared components repo, we don't manually update the version. We let Lerna do the magic in which when we publish the package (after PR has been merged), Lerna will automatically update the version (Sev is the one that has access to do the publishing).

**Comment by JazzarKarim** [comment]
Don't forget to rebase please 🙌

File: src/components/approval-type/package.json (hunk 1 above)
Comment (PaulGarewal) [reply src/components/approval-type/package.json:3]: Ok I'll change that back

Comment (JazzarKarim) [reply src/components/approval-type/package.json]: Thanks! Resolving.

File: src/components/approval-type/ApprovalType.vue [hunk 2]
```diff
@@ -20,12 +20,12 @@
...
           <!-- COURT ORDER section -->
           <template v-if="!isCourtOrderRadio">
-            <span class="v-label ml-2">{{ getRadioText(ApprovalTypes.VIA_COURT_ORDER) }}</span>
+            <span class="v-label ml-2 mb-2">{{ getRadioText(ApprovalTypes.VIA_COURT_ORDER) }}</span>
           </template>
           <template v-else>
             <v-radio
               id="court-order-radio"
-              class="mb-0"
+              class="mb-n3"
```
Comment (severinbeauvais) [inline src/components/approval-type/ApprovalType.vue:28]: Could you provide screenshots of what both these changes do, please? You should be able to run the component in Storybook and take snaps of that... or use this component right in the app (you can temporarily change it in situ in the node_modules folder).

File: src/components/approval-type/package.json (hunk 1 above)
Comment (severinbeauvais) [reply src/components/approval-type/package.json]: The following people have admin permissions in bcrs-shared-components and can run the Lerna commands:
Argus
SRE (shared login)
Kial
Meng
Omid
Patrick
Richard
Travis
Severin
Thor
Vysakh

**Pull request merged**